provider/provider.go

```diff
@@ -226,7 +229,7 @@ func configureDatabricksClient(ctx context.Context, d *schema.ResourceData) (any
 		}
 	}
 	sort.Strings(attrsUsed)
-	log.Printf("[INFO] Explicit and implicit attributes: %s", strings.Join(attrsUsed, ", "))
+	tflog.Info(ctx, fmt.Sprintf("Explicit and implicit attributes: %s", strings.Join(attrsUsed, ", ")))
```

Comment:
This should be a no-op, but I needed to make this change so that I could enable logging for just our provider using `TF_LOG_PROVIDER_DATABRICKS=INFO` (which according to the docs, only enables `tflog.*` logs and not `log.Printf` logs). In general, however, we should be using `tflog.*` everywhere for logging to fit into TF's logging framework.